app/schemas/models/exam.schema.js

```diff
@@ -0,0 +1,59 @@
+const c = require('./../schemas')
+
+const ExamSchema = c.object({
+  title: 'Exam',
+  description: 'CodeCombat exam with limited levels and complete time',
+})
+
+_.extend(ExamSchema.properties, {
+  title: {
+    type: 'string',
+    description: 'Title of the exam',
+  },
+  description: {
+    type: 'string',
+    description: 'Description of the exam',
+  },
+  clan: c.objectId({ links: [{ rel: 'db', href: '/db/clan/{($)}' }] }),
+  startDate: c.stringDate(),
+  endDate: c.stringDate(),
+  duration: {
+    type: 'number',
+    description: 'Duration of the exam in minutes',
+  },
+  problems: {
+    type: 'object',
+    description: 'Problems that are included in the exam',
+    additionalProperties: {
+      type: 'object',
+      title: 'ExamProblemCourse',
+      description: 'course and instance',
+      // key is the courseID
+      properties: {
+        instanceId: {
+          type: 'string',
+          description: 'ID of the course instance',
+        },
+        levels: {
+          type: 'array',
+          description: 'Levels that are included in the exam',
+          items: c.object({
+            title: 'ExamProblemLevel',
+            description: 'Level that is included in the exam',
+            slug: {
+              type: 'string',
+              description: 'Slug of the level',
+            },
+            point: {
```

Comment:
i dont understand what this field does

Reply:
in case we do not want to every level has same points in the exam. (I don't know is it needed, we can discuss) . for example 10 cs1 levels which have 1 point each level, 30 cs4 levels which has 3 points each level, and summary to 100.

Reply:
haha, we dont need it right now. But I'm ok with having it but let's add a proper description for it like your comment here.